optimum/habana/transformers/training_args.py

```diff
@@ -115,6 +115,14 @@ class GaudiTrainingArguments(TrainingArguments):
             Number of steps to ignore for profling.
         profiling_steps (`int`, *optional*, defaults to 0):
             Number of steps to be captured when enabling profiling.
+        profiling_record_shapes (`boolean`, *optional*, defaults to True):
```

Comment:
Where are we using it? Default value is taken from original transformers?

Reply:
This has nothing to do with current change. I just added a missing comment for PT profiler. Its being used in PT profiler initialization in trainer.py, it is habana specific argument (not there in original transformers).